### DISCUSSION TITLE
mistake at fluid provider station

### DISCUSSION THREAD

**Discussion by hytrychort** [Opinionated-Blueprints/10-Books-Full-of-Rails#27] [Q&A] [answered]
there is a mistake in the liquid provider station: in a constant combinator, the value of steel chests should be 100k, not 200k. Because of this error, trains do not travel to the provider's station

**Comment by Bocian-1** [comment, marked as answer]
This value is calculated in blueprint parametrisation as 50k (volume of vanilla fluid wagon) * number of fluid wagons. If you set the number of fluid wagons to 4 when placing the blueprint, it will be 200k.
